README.md

```diff
@@ -1 +1,32 @@
-# self-development
\ No newline at end of file
+# Саморазвитие
+Ниже представлен список рекомендаций участников сообщества it-muslim, который призван сэкономить ваше время и силы в поиске достойных ресурсов для самообучения. Список разбит на тематики, каждая рекомендация сопровождается одним или несколькими отзывами. В случае, если рекомендация доступна только на английском или платно, это помечается тэгами `eng` и `$` соответственно. 
+
+## Содержание
+- [Основы программирования](#основы-программирования)
+- [HTML & CSS](#html--css)
+- [JavaScript](#javascript)
+- [TypeScript](#typescript)
+- [Базы данных](#базы-данных)
+- [Soft skills](#soft-skills)
+- [Contributing](#contributing)
+
+## Основы программирования
```

Comment:
Let's make all headers in English, so our readers can get accustomed to English terminology.